interview-questions.md

```diff
@@ -8,62 +8,68 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: a parameter is a place holder for information. An argument is part of a function
 
+  Researched answer: An argument is the information of a function, the parameters are place holders for the arguments.
```

Comment:
This is good. Is this what you want to present in an interview though? You're not wrong but will this win you the points you need for a job. Feel free to write about your experience so you have that on the books ready to go.